tests/suites/test_suite_ssl.function

```diff
@@ -12,6 +12,48 @@
 
 #define SSL_MESSAGE_QUEUE_INIT      { NULL, 0, 0, 0 }
 
+#if defined(MBEDTLS_SSL_EARLY_DATA) && defined(MBEDTLS_SSL_CLI_C) && \
```

Comment:
Should this list start with `PROTO_TLS1_3`? Or is that implied by the others? I'll note `tls13_resume_session_with_ticket` has an explicit dependency on `PROTO_TLS1_3`.

Reply:
MBEDTLS_SSL_EARLY_DATA => MBEDTLS_SSL_PROTO_TLS1_3 thus we do not need MBEDTLS_SSL_PROTO_TLS1_3 here but in the other function yes.